xla/service/cpu/ir_emitter.cc

```diff
@@ -2730,6 +2730,16 @@ absl::Status IrEmitter::HandleOneDnnLayerNorm(HloInstruction* custom_call) {
 }
 
 absl::Status IrEmitter::HandleOneDnnSoftmax(HloInstruction* custom_call) {
+  // OneDnnSoftmaxConfig.
```

Comment:
Can you explain the purpose of this comment? It is either not expansive enough (in which case, please elaborate it), or not useful (in which case, please remove it).